contractsV5/KyberNetwork.sol

```diff
@@ -266,20 +266,27 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
         returns (uint expectedRate, uint worstRate)
     {
         if (src == dest) return (0, 0);
+        uint permHintGetRate = 1 << 255; //for backwards compatibility
+        uint qty = ((srcQty & permHintGetRate) > 0) ? srcQty & ~permHintGetRate : srcQty;
```

Comment:
I think just turn the bit off.
and avoid extra code.
unless it saves gas the way its don't here. which I don't see why it should.

Reply:
20 gas cheaper with global constant